x-pack/plugins/upgrade_assistant/public/application/components/es_deprecation_logs/fix_deprecation_logs/fix_logs_step.tsx

```diff
@@ -126,15 +116,6 @@ const FixLogsStep: FunctionComponent<Props> = ({
     saveLogsCheckpoint(checkpoint);
   }, [checkpoint]);
 
-  useEffect(() => {
-    if (!isDeprecationLogIndexingEnabled) {
-      setIsComplete(false);
-    }
-
-    // Depending upon setIsComplete would create an infinite loop.
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [isDeprecationLogIndexingEnabled]);
```

Comment:
This was part of the logic used used for setting the step completion when there was no deprecation logs.